egs/gigaspeech/ASR/zipformer/export.py

```diff
@@ -0,0 +1,522 @@
+#!/usr/bin/env python3
```

Comment:
Can you replace it with a symlink?

Reply:
I changed some comments like huggingface link in it.